corehq/apps/custom_data_fields/models.py

```diff
@@ -40,6 +40,8 @@ class CustomDataField(JsonObject):
     label = StringProperty()
     choices = StringListProperty()
     is_multiple_choice = BooleanProperty(default=False)
+    # Currently only relevant for location fields
+    index_in_fixture = BooleanProperty(default=False)
```

Comment:
nit: Can we add comment to share what happen when this is set to true. I don't think index_in_fixture is able to tell exactly what this would do unless someone understands fixtures well.

Reply:
I just added some descriptions to the feature flag and the UI

Reply:
Cool. We should write a note to refer to the FF then. Worried though in case the FF is removed in future, the context would go.

Reply:
What kind of context are you looking for?  This flag tells whether to index the field in the restore, so I thought `index_in_fixture` describes it pretty well.  The UI where this flag is turned on also has help text regarding the broader implications of toggling this.